mix.exs

```diff
@@ -37,7 +37,9 @@ defmodule Staccato.Mixfile do
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
-      {:httpoison, "~> 1.2"}
+      {:poison, "~> 3.1"},
+      {:httpoison, "~> 1.2"},
+      {:elixir_uuid, "~> 1.2"}
```

Comment:
I would normally resist adding this for something as straightforward. UUID generation isn't likely to change, but this library will be something I'll have to update. I could go either way having looked at the library itself.

Reply:
Elixir-UUID is fairly widely adopted from what I've seen, why reinvent the wheel?

Reply:
That said, totally fair point about bringing simple things as dependencies (I totally try to do the same normally).

Reply:
That sounds fine. If you feel like it's well adopted, let's use it.

Reply:
👍